### ISSUE TITLE
Accessibility: use `only_for_screenreader` instead of `None` for labels

### ISSUE THREAD

**Issue by mccalluc** [opendp/dp-wizard#621]
We don't always fill in the `label` param for inputs.
- The Shiny label is wrapped to the width of the input
- It may be clear from context

But for accessibility, we should be filling labels in. This PR
- #618

adds a `only_for_screenreader()` function that we can use.
